<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDlg</class>
 <widget class="QDialog" name="NewProjectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Project &amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>edName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edName">
       <property name="maxLength">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Project &amp;Location:</string>
       </property>
       <property name="buddy">
        <cstring>edPath</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edPath"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>&amp;Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="btnGroup">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnGroup</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGroup</sender>
   <signal>rejected()</signal>
   <receiver>NewProjectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
